images/neuvector/tests/internal-certs.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Install openssl
+apk add openssl openssl-config
+
+# Generate CA
+openssl genrsa -out ca.key 20485
```

Comment:
That's a pretty decent sized key you've got there..

Reply:
Typo haha. Also forgot the ca config...